cloudvolume/secrets.py

```diff
@@ -113,6 +113,14 @@ def aws_credentials(bucket = '', service = 'aws'):
       with open(aws_credentials_path, 'r') as f:
         aws_credentials = json.loads(f.read())
       break
+  
+  if not aws_credentials:
```

Comment:
I don't think I understand the use case here. When would an environment variable be present but not the file? I do think we should support CloudVolume accepting credentials in the constructor including the s3 endpoint per #170 Does this introduce any security issues that were not present with the files?

I guess AWS thinks it's ok: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html

Reply:
environment variable could be transferred by `docker run -e`, and also defined in the kubernetes yml file. 
also boto3 and awscli will load environment variable too, so we only need to define environment variable and all the aws tools will work.